### PR TITLE
I made text non static for no reason.

### PR DESCRIPTION
**Pull request by nobody5050**
At least I think I did. For all I know I could have forgotten a file somewhere in the web of typescript that is this project. If it is able to be nonstatic then I must point out there might not be explicit support for this in the actual game. In that case open source the game an I’ll make a pull request there too :p

### CODE REVIEW

File: src/components/TextParam.tsx [hunk 1]
```diff
@@ -94,6 +94,7 @@ const ParamsBox: FunctionComponent = () => {
 				<button onClick={onAddLanguage}>Add language</button>
 			</div>
 		</Fragment>
+		<label>static: <input type="checkbox" checked={selectedEntity.params.isStatic} onChange={onToggleStatic}/></label>
```
Comment (getkey) [inline src/components/TextParam.tsx:97]: No need for this, `<StaticParam/>` already does it.

**Review by getkey — COMMENTED** [review]
Haha this works. But indeed, the game doesn't support it. It would just ignore it.

File: src/components/TextParam.tsx (hunk 1 above)
Comment (nobody5050) [reply src/components/TextParam.tsx:97]: Alright, I wasn’t sure if it would be needed. Does this just add an extra static checkbox?

Comment (getkey) [reply src/components/TextParam.tsx:97]: Yep. And it would break when you click it because `onToggleStatic` doesn't exist.

Comment (nobody5050) [reply src/components/TextParam.tsx:97]: oh yeah since it’s in another file.

**Comment by nobody5050** [comment]
I’m so happy that for the most part what I changed wouldn’t break anything

**Comment by nobody5050** [comment]
closing since it was a joke

**Pull request closed without merging**